docs/src/examples/components/Avatar/Variations/AvatarExampleStatus.shorthand.tsx

```diff
@@ -4,37 +4,37 @@ import { Avatar } from '@stardust-ui/react'
 const AvatarExampleStatusShorthand = () => (
   <div style={{ background: 'white' }}>
     <Avatar
-      src="/public/images/avatar/small/matt.jpg"
+      src="public/images/avatar/small/matt.jpg"
       status="Available"
       style={{ marginRight: '15px' }}
```

Comment:
Inline styles should be replaced here.  Either leave the avatars inline or use something like `&emsp;` in between them to achieve spacing.